docs/latest/python/python-sources.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: docs
+title:  Creating source streams in Python
+description:  Learn how to develop a source stream in Python by using the the Python Application API
+---
+
+**Important:** The Python Application API is currently an alpha release.
+
+# Python sources
+
+## Overview
+
+The function `Topology.source` declares a source stream, one that brings external data into your Streams application.
+A source stream is the start of a streaming graph.
+
+The `source` function is passed an application function that returns an [iterable](https://docs.python.org/3/glossary.html#term-iterable).
```

Comment:
I would just add that an "application function" is a function which is defined in a non-main module.